.ci/docker/almalinux/Dockerfile

```diff
@@ -1,33 +1,27 @@
-ARG CUDA_VERSION=10.2
+ARG CUDA_VERSION=12.4
 ARG BASE_TARGET=cuda${CUDA_VERSION}
-FROM centos:7 as base
```

Comment:
Shouldn't base image be an argument then? So that we can still build old ones, if needed as well as new oens?

Reply:
Oh, wow, I see, you did `git cp`, was a bit confused by the pattern